apps/web/app/(app)/environments/[environmentId]/components/MainNavigation.tsx

```diff
@@ -90,6 +90,14 @@ export const MainNavigation = ({
     localStorage.setItem("isMainNavCollapsed", isCollapsed ? "false" : "true");
   };
 
+  useEffect(() => {
+    if (typeof window !== "undefined") {
```

Comment:
Since a useEffect is always running in client / browser, do we need this check here or can we also simplify this by removing the window check?

Reply:
yeah, makes sense 👍